Directory.Build.targets

```diff
@@ -7,4 +7,13 @@ https://github.com/dotnet/sourcelink/issues/572 -->
   <ItemGroup>
     <EmbeddedFiles Include="$(GeneratedAssemblyInfoFile)"/>
   </ItemGroup>
+  
+  <Target Name="CoverletGetPathMap"
```

Comment:
Are you missing?
```xml
  <ItemGroup>
    <SourceRoot Include="$(NuGetPackageRoot)" />
  </ItemGroup>
```

Reply:
Shouldn't need that. That workaround is only needed for packages that contain source files.